ramls/loan.json

```
@@ -242,6 +242,11 @@
       },
       "additionalProperties": false
     },
+    "itemEffectiveLocationIdAtCheckOut": {
```

Is it important that the location was derived in an internal process within inventory? As far as circulation is concerned, the item's location is the effective location. Just a thought.

```suggestion
    "itemLocationIdAtCheckOut": {
```

---

@marcjohnson-kint You had previously suggested using the word "effective", based on Nassib's description, "The effective location, at the time of checkout, of the item loaned to the patron". I can remove the word, if you want.
https://github.com/folio-org/mod-circulation-storage/pull/194#discussion_r324739462

---

@rjruizes No, don't worry, I've gone back and forth on this. I have a feeling folks will find it confusing without effective, even though I think this is excessive bleeding of the inventory domain into circulation